src/client/owner.py

```diff
@@ -1,19 +1,19 @@
-__all__ = ['Owner']
-from .primitives.base import BaseTolokaObject
-
-
-class Owner(BaseTolokaObject):
-    """Parameters of the customer who created an object.
-
-    Attributes:
-        id: Customer ID.
-        myself: An object accessory marker.
-            Possible values:
-                * True - an object created by the customer whose OAuth-токен in the request;
-                * False - an object does not belong to the customer whose OAuth-токен in the request.
-        company_id: ID of the customer's company.
-    """
-
-    id: str
-    myself: bool
-    company_id: str
+__all__ = ['Owner']
+from .primitives.base import BaseTolokaObject
+
+
+class Owner(BaseTolokaObject):
+    """Parameters of the customer who created an object.
+
+    Attributes:
+        id: Customer ID.
+        myself: An object accessory marker.
+            Possible values:
+                * `True` — An object is created by the customer whose OAuth-token is used in the request.
+                * `False` — An object does not belong to the customer whose OAuth-token is used in the request.

```

Comment:
No need for a hyphen here, should be `OAuth token`

Reply:
Fixed